bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java

```diff
@@ -106,13 +116,24 @@ public HBaseRequestAdapter(BigtableOptions options,
   HBaseRequestAdapter(TableName tableName,
                               BigtableTableName bigtableTableName,
                               MutationAdapters mutationAdapters) {
+    this(tableName, bigtableTableName, mutationAdapters, false);
+  }
+
+  private HBaseRequestAdapter(TableName tableName,
+      BigtableTableName bigtableTableName,
+      MutationAdapters mutationAdapters, boolean allowServersideTimestamp) {
```

Comment:
Please add a precondition to ensure that MutationAdapters and allowServersideTimestamp don't go out of sync

Reply:
Or better yet, don't duplicate it and ask MutationAdapters

Reply:
Should we add package private getter in PutAdapter to get this information or add it in MutationAdapters?

Reply:
Probably private getter